.github/ci/files/models/DataObject/Fieldcollection/Data/FilterCategory.php

```diff
@@ -30,45 +30,48 @@ class FilterCategory extends \Pimcore\Bundle\EcommerceFrameworkBundle\Model\Cate
 * Get label - Label
 * @return string|null
 */
-public function getLabel (): ?string {
+public function getLabel(): ?string {
 	$data = $this->label;
 	if ($data instanceof \Pimcore\Model\DataObject\Data\EncryptedField) {
-		    return $data->getPlain();
+		return $data->getPlain();
 	}
-	 return $data;
+	return $data;
 }
 
 /**
 * Set label - Label
 * @param string|null $label
 * @return \Pimcore\Model\DataObject\Fieldcollection\Data\FilterCategory
 */
-public function setLabel (?string $label) {
+public function setLabel(?string $label) {
+	/** @var \Pimcore\Model\DataObject\ClassDefinition\Data\Input $fd */
 	$fd = $this->getDefinition()->getFieldDefinition("label");
```

Comment:
Actually, there's no need to generate this line in this case at all, as it is never used.

Reply:
@jdreesen I removed the unused field definition variables